### DISCUSSION TITLE
Опять скачиваются пустые папки

### DISCUSSION THREAD

**Discussion by kitty-chan2022** [lirix360/ReadmangaGrabber#70] [Q&A]
Я скачивала мангу https://mintmanga.live/eromanga_sensei__A5664 , но папки пустые. Я пробовала скачивать другие манги, они скачиваются, именно с этой мангой что-то не так. Почему?

**Comment by lirix360** [comment]
Программа на данный момент не поддерживает скачивание манги требующей авторизации для просмотра на сайте.
